chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -1212,33 +1206,22 @@ impl PeerManagerActor {
         );
     }
 
-    fn verify_edges(&mut self, ctx: &mut Context<Self>, peer_id: PeerId, edges: Vec<Edge>) {
+    fn verify_edges(&mut self, _ctx: &mut Context<Self>, peer_id: PeerId, edges: Vec<Edge>) {
```

Comment:
can you mention what this method does?

as it seems to be banning peers that send the invalid edges - but I don't know what happens to the invalid edges themselves. Are we removing them somewhere from the list?

Reply:
https://github.com/near/nearcore/pull/5429